src/RadixDlt.NetworkGateway.GatewayApi/Validators/EntityDetailsRequestValidator.cs

```diff
@@ -67,15 +67,24 @@
 
 namespace RadixDlt.NetworkGateway.GatewayApi.Validators;
 
-internal class EntityDetailsRequestValidator : AbstractValidator<GatewayModel.EntityDetailsRequest>
+internal class EntityDetailsRequestValidator : AbstractValidator<GatewayModel.StateEntityDetailsRequest>
 {
     public EntityDetailsRequestValidator(LedgerStateSelectorValidator ledgerStateSelectorValidator)
     {
-        RuleFor(x => x.Address)
-            .NotEmpty()
-            .RadixAddress();
-
         RuleFor(x => x.AtLedgerState)
             .SetValidator(ledgerStateSelectorValidator);
+
+        RuleFor(x => x.Addresses)
+            .NotEmpty()
+            .DependentRules(() =>
+            {
+                RuleFor(x => x.Addresses.Count)
+                    .GreaterThan(0)
+                    .LessThan(5);
```

Comment:
thought we wanted to support only 1 address for now?

Reply:
Reconfigured for exactly 1.